core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/transmit/cache.go

```diff
@@ -28,7 +28,7 @@ func (c *transmitEventCache) get(block ocr2keepers.BlockNumber, logID string) (o
 	c.lock.RLock()
 	defer c.lock.RUnlock()
 
-	i := int64(block) % int64(c.cap)
+	i := int64(block) % c.cap
 	b := c.buffer[i]
 	if b.block != block {
 		return ocr2keepers.TransmitEvent{}, false
```

Comment:
/nit This feels more readable to me

```
func (c *transmitEventCache) get(block int64, logID string) (ocr2keepers.TransmitEvent, bool) {
	c.lock.RLock()
	defer c.lock.RUnlock()

	i := block % c.cap // no longer has an int64 cast on block
	b := c.buffer[i]
	if int64(b.block) != block { //has an int64 cast on b.block
		return ocr2keepers.TransmitEvent{}, false
	}
	if len(b.records) == 0 {
		return ocr2keepers.TransmitEvent{}, false
	}
	e, ok := b.records[logID]

	return e, ok
}
```
I'm not sure on the tradeoff with this pattern:
`type BlockNumber uint64`

What's the readability/usability gain relative to all the casting when we need to perform an int64 operation?

Reply:
I don't know this code, but that makes sense to me.